assets/css/general.css

```diff
@@ -223,12 +303,39 @@ html[dir='rtl'] .clearButton {
     z-index: 10000000000000;
 }
 
+/* FontAwesome and Bootstrap 4 helpers */
+.fa.float-right { /* sass-lint:disable-line class-name-format */
+    margin-left: .3em;
+}
+
 /* RTL checking */
 
-html[dir='rtl'] .pull-right:not(#rtlChecker) { /* sass-lint:disable-line class-name-format */
+html[dir='rtl'] body {
+    text-align: right !important; /* sass-lint:disable-line no-important */
+}
+
+html[dir='rtl'] .float-right:not(#rtlChecker) { /* sass-lint:disable-line class-name-format */
     float: left !important; /* sass-lint:disable-line no-important */
 }
 
-html[dir='rtl'] .pull-left:not(#rtlChecker) { /* sass-lint:disable-line class-name-format */
+html[dir='rtl'] .float-left:not(#rtlChecker) { /* sass-lint:disable-line class-name-format */
     float: right !important; /* sass-lint:disable-line no-important */
 }
+
+html[dir='rtl'] label input[type='checkbox'] {
+    margin-right: -20px;
+    margin-top: 5px;
+}
+
+html[dir='rtl'] button.close { /* sass-lint:disable-line no-qualifying-elements */
+    margin-left: 0 !important; /* sass-lint:disable-line no-important */
+}
+
+html[dir='rtl'] .btn-group > .btn { /* sass-lint:disable-line class-name-format */
+    border-radius: .2rem 0 0 .2rem;
+}
+
+html[dir='rtl'] .btn-group > .btn:first-child:not(.dropdown-toggle), /* sass-lint:disable-line class-name-format */
```

Comment:
What does this actually apply to? If it's the translation dropdown bits, can they be moved? Furthermore, I think there's a classname you could add here that would make it less general. Similar concern below.

Reply:
Done i think, Can you please have a look is it alright?